app/models/form_submission.rb

```diff
@@ -11,36 +11,16 @@ class FormSubmission < ApplicationRecord
   validates :form_type, presence: true
 
   class << self
-    # rubocop:disable Metrics/MethodLength
     def with_latest_benefits_intake_uuid(user_account)
-      # This query retrieves distinct form submissions for a given user account,
-      # along with the latest benefits intake UUID for each form. If a form has any
-      # submission attempts with a benefits_intake_uuid, it uses the latest attempt's UUID;
-      # otherwise, it falls back to the form submission's original UUID.
-      FormSubmission
-        .select(
-          'DISTINCT form_submissions.id,
-          form_submissions.form_type,
-          COALESCE(latest_attempts.benefits_intake_uuid, form_submissions.benefits_intake_uuid) AS benefits_intake_uuid,
-          form_submissions.created_at'
-        )
-        .left_joins(:form_submission_attempts)
+      select('form_submissions.id, form_submissions.form_type, la.benefits_intake_uuid, form_submissions.created_at')
```

Comment:
do we need any additional tests to be confident this new change has parity with the old version? or the below tests are sufficient?

Reply:
This change was added by @tpharrison and I think the below tests will be sufficient.